components/transfer/TransferConfirmModal.vue

```diff
@@ -113,11 +114,17 @@
                         :address="address.address"
                         hide-identity-popover />
                     </span>
-                    <NeoIcon
-                      icon="circle-info"
-                      class="is-size-6"
-                      pack="far"
-                      :title="address.address" />
+                    <NeoTooltip
+                      :label="address.address"
+                      append-to-body
+                      :content-class="`transfer-tooltip ${
+                        isMobile ? 'mobile' : ''
+                      }`">
```

Comment:
Is the classname `mobile` used somewhere?
```suggestion
                      content-class="transfer-tooltip">
```